### ISSUE TITLE
Can not access web page after updating system.

### ISSUE THREAD

**Issue by JadinTredup** [mindaffect/General#1]
Hello, I have recently been having difficulties setting up my decoder to run on my Wifi network so that I could update it. It finally decided to connect today and so I pressed the update button. Now, whenever I start the decoder, I can not find its IP address using `arp -a` and it is not creating its own hotspot network.

**Comment by jadref** [comment]
Hi Jadin,  I'm sorry to hear that.  Can you attached a screen to see if it is booting correctly and where it is connecting to?  If it is booking correctly then it will also show a window on the screen with the IP information of the host.

Jason

**Comment by JadinTredup** [comment]
Hi Jason, thanks for responding so quickly. So I looked deeper into this and I figured out what happened and it was my fault. I was incorrect in that it was connecting to my wifi. I accidentally entered my Wifi credentials in the field for setting a custom SSID for the decoder hotspot and so I was mimicking my own wifi network with the decoder. It appears the update did not set in and I couldn't figure out how to reset the Hotspot network without accessing it so I reflashed the SD image.
